util/topsql/reporter/reporter.go

```diff
@@ -453,8 +496,6 @@ func (tsr *RemoteTopSQLReporter) takeDataAndSendToReportChan(collectedDataPtr *m
 	select {
 	case tsr.reportCollectedDataChan <- data:
 	default:
-		// ignore if chan blocked
-		ignoreReportChannelFullCounter.Inc()
```

Comment:
Why remove this metric?

Reply:
Nothing will block `reportCollectedDataChan` to receive data so it's hard to be full. But I can put it back if you want.